package.json

```diff
@@ -24,7 +24,7 @@
     "gulp-scss-lint": "0.4.0",
     "gulp-util": "3.0.7",
     "handlebars": "^4.0.5",
-    "metalsmith": "^2.1.0",
+    "metalsmith": "^1.7.0",
```

Comment:
The purpose of this PR is just to kick off a Travis build - downgraded metalsmith was one recommended step but it doesn't seem to have resolved it anyway.